src/pages/what-is-ethereum.tsx

```diff
@@ -320,524 +295,560 @@ const WhatIsEthereumPage = ({
-        </Section>
-
-        <Section>
-          <TwoColumnContent direction={{ base: "column", lg: "row-reverse" }}>
-            <Width40>
-              <Image400 src={whatAreSmartContracts} />
-            </Width40>
-            <Width60>
-              <H2>{t("page-what-is-ethereum-smart-contract-title")}</H2>
-              <Text>
-                <Translation id="page-what-is-ethereum:page-what-is-ethereum-smart-contract-desc-1" />
-              </Text>
-              <Text>
-                <Translation id="page-what-is-ethereum:page-what-is-ethereum-smart-contract-desc-2" />
-              </Text>
-              <Text>
-                <Translation id="page-what-is-ethereum:page-what-is-ethereum-smart-contract-desc-3" />
-              </Text>
-              <ButtonRow>
-                <ButtonLink href="/smart-contracts/">
-                  {t("page-what-is-ethereum-more-on-smart-contracts")}
-                </ButtonLink>
-                <ButtonLink href="/dapps/" variant="outline" isSecondary>
-                  {t("page-what-is-ethereum-explore-dapps")}
-                </ButtonLink>
-              </ButtonRow>
-            </Width60>
-          </TwoColumnContent>
-        </Section>
-
-        <Section bgColor="homeBoxTurquoise">
-          <TwoColumnContent>
-            <Width40>
-              <Image400 src={ethCoin} />
-            </Width40>
-            <Width60>
-              <H2>{t("page-what-is-ethereum-meet-ether-title")}</H2>
-              <Text>{t("page-what-is-ethereum-meet-ether-desc-1")}</Text>
-              <Text>{t("page-what-is-ethereum-meet-ether-desc-2")}</Text>
-              <ButtonRow>
-                <ButtonLink href="/eth/">
-                  {t("page-what-is-ethereum-what-is-ether")}
-                </ButtonLink>
-                <ButtonLink href="/get-eth/" variant="outline" isSecondary>
+            </Hero>
+          </Flex>
+        </Content>
+        <div className="w-full bg-[#fcfcfc] md:shadow-[inset_0px_1px_0px_rgba(0,0,0,0.1)] dark:bg-[#272627] dark:md:shadow-[inset_0px_1px_0px_hsla(0,0%,100%,.1)]">
+          <Section>
+            <Stack className="gap-14">
+              <TwoColumnContent id="summary">
+                <Width60>
+                  <Summary>
+                    <Stack className="gap-6">
+                      <h2 className="text-[1.4rem] leading-xs text-[#4c4c4c] dark:text-[#ccc]">
+                        {t("page-what-is-ethereum-summary-title")}
+                      </h2>
+                      <Stack className="gap-[1.45rem]">
+                        <p>{t("page-what-is-ethereum-summary-desc-1")}</p>
+                        <p>{t("page-what-is-ethereum-summary-desc-2")}</p>
+                        <p>{t("page-what-is-ethereum-summary-desc-3")}</p>
+                      </Stack>
+                    </Stack>
+                  </Summary>
+                </Width60>
+                <Width40 />
+              </TwoColumnContent>
+              <div>
```

Comment:
This was a missing space we needed to add
```suggestion
              <Stack className="gap-8">
```